drawio/automation/requirements.txt

```diff
@@ -0,0 +1,3 @@
+csv
+N2G # https://n2g.readthedocs.io/en/latest/diagram_plugins/DrawIo%20Module.html
+python-igraph
```

Comment:
might want to pin the versions of these. Save yourself a potential headache later.